modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,22 +12,51 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController'
       })
       .state('articles.create', {
         url: '/create',
         templateUrl: 'modules/articles/client/views/create-article.client.view.html',
+        controller: 'ArticlesController',
```

Comment:
Shouldn't this be 'ArticlesCreateController' or something similar since we are doing that above?